api/utils/doc-to-account.js

```diff
@@ -16,12 +16,9 @@ function toAccount (doc, options) {
   var account = {
     id: accountId,
     username: username,
+    profile: doc.profile || {},
     roles: findCustomRoles(roles)
   }
 
-  if (options.includeProfile) {
-    account.profile = doc.profile
-  }
```

Comment:
I would leave that in, but replace `account.profile = doc.profile` with `account.profile = doc.profile || {}`